cfep-18.md

```diff
@@ -0,0 +1,90 @@
+This means that executables that are statically linked also need to include the license headers of most of their libraries.
+
+Using dynamic linkage also has the benefit of a smaller disk usage even when the static archives are not installed, 
+as we will have the symbols of a library copied into every binary that uses a library.
+In the dynamic setting, the symbols would only exist once on disk, inside the shared object.
+
+Not only does dynamic linkage reduce the storage requirements on disk but it also reduces the memory usage when several processes use the same library.
+Depending on the operation system, the symbols are then only loaded once into memory and shared between the processes.
+
+Building static archives (in separated packages) is still useful for some non-common workflows.
+These don't represent the majority of uses of our packages but can profit a lot from our using our existing infrastructure.
+
+## Implementation
+
+To implement the CFEP, we would remove static libraries from existing packages in new builds.
+Quite often static libraries were included by default without the need for them or just because they are the default output.
+They are in most cases not used/needed.
+
+If the static libraries are needed though, they should be added as a separate output with a `-static` suffix, see https://github.com/conda-forge/krb5-feedstock/pull/23/files .
+
+Packages that statically link a dependency need to take care that they include the license file of all dependencies that require it.
+For now, we will also don't do any automatic rebuilds for packages with static linkage.
+This means that packages using static linkage need to regularly rebuild to include the updated versions.
+Otherwise the end-user experience will be that the library wasn't updated at all in the stack.
+
+## Windows
+
+We are going to be less strict on Windows builds for static libraries, mainly as the support for builds on Windows is less mature for the majority of tools.
+There are a lot libraries that only work with static linkage as the upstreams didn't take care to define the correct `__dllexport/__dllimport` lines.
+If a build works fine with shared linkage, it should only providing those.
+If a build doesn't yet work with shared linkage on Windows, we shouldn't force the maintainer of the feedstock to write a huge patch to make the linkage work.
+There is already a lot of hassle and not that much community support for packaging things on Windows.
+
+One additional caveat on Windows is though that there are several types of static libraries.
+In addition to every shared library (`bin\<name>.dll`) there is always accompanying (static) import library (`lib\<name>.lib`).
+As the import libraries are required for the linker to work, they definitely need to be included in the default installation.
+Sometimes the static build of a library has the same name as the import library, sometimes it comes with a `_static` suffix, i.e. `lib\<name>_static.lib`.
```

Comment:
I have also seen the `_a` suffix (`<name>_a.lib`)

I am wondering if this CFEP should include a preferred naming scheme for static libs on Windows?